prow/README.md

```diff
@@ -40,6 +40,16 @@ Note: versions specified in these announcements may not include bug fixes made
 in more recent versions so it is recommended that the most recent versions are
 used when updating deployments.
 
+ - *April 2, 2018* `updateconfig` format has been changed from
+   ```yaml
+   path/to/some/other/thing: configName
+   ```
+   to
+   ```yaml
+   path/to/some/other/thing:
+     Name: configName
+     Namespace: nonDefaultNamespace
```

Comment:
This is misleading. It suggests that the namespace must be something other than "default",  but in reality it should be something other than the value of `ProwJobNamespace` (which may or may not be "default").

Reply:
Let me know what you think